app/src/main/java/org/simple/clinic/home/patients/PatientsEffectHandler.kt

```diff
@@ -58,7 +58,7 @@ class PatientsEffectHandler @AssistedInject constructor(
         .addTransformer(LoadNumberOfPatientsRegistered::class.java, loadNumberOfPatientsRegistered())
         .addTransformer(LoadInfoForShowingAppUpdateMessage::class.java, loadInfoForShowingAppUpdate())
         .addConsumer(TouchAppUpdateShownAtTime::class.java, { appUpdateDialogShownAtPref.set(Instant.now(utcClock)) }, schedulers.io())
-        .addConsumer(PatientsTabViewEffect::class.java, viewEffectHandler::handle, schedulers.ui())
+        .addConsumer(PatientsTabViewEffect::class.java, viewEffectsConsumer::accept, schedulers.ui())
```

Comment:
How would this work? Would accept function be internally calling handle?

Reply:
Mobius will take care of that, if you look at the `BaseScreen`, we are observing the view effects live queue and dispatching them to our view effect handler. The live queue we are observing is a lifecycle aware component.

Reply:
You can refer this to learn more: https://github.com/spotify/mobius/pull/78